themes/agnoster.zsh-theme

```diff
@@ -109,7 +109,9 @@ prompt_git() {
   if $(git rev-parse --is-inside-work-tree >/dev/null 2>&1); then
     repo_path=$(git rev-parse --git-dir 2>/dev/null)
     dirty=$(parse_git_dirty)
-    ref=$(git symbolic-ref HEAD 2> /dev/null) || ref="➦ $(git rev-parse --short HEAD 2> /dev/null)"
+    ref=$(git symbolic-ref HEAD 2> /dev/null) || \
+    ref=" $(git describe --exact-match HEAD 2> /dev/null)" ||\
```

Comment:
The unknown character is `nf-fa-tag` (`f02b`). Working in the command line when the powerline fonts are installed.

Reply:
That is not actually a Powerline font glyph, but a Font Awesome one. Powerline font glyphs are only the ones starting with `nf-pl-` in https://www.nerdfonts.com/cheat-sheet. We should use a different glyph, preferably one in the UTF-8 spec because Powerline glyphs are very few.
Also, to keep this consistent, there should be a space between `||` and `\`:
```suggestion
    ref="◈ $(git describe --exact-match HEAD 2> /dev/null)" || \
```

Reply:
Also, the proper command is `git describe --exact-match --tags HEAD`, otherwise non-annotated tags aren't matched:
```suggestion
    ref="◈ $(git describe --exact-match --tags HEAD 2> /dev/null)" ||\
```